hydromt_wflow/wflow.py

```diff
@@ -2759,7 +2764,12 @@ def setup_lulcmaps_with_paddy(
                 v: f"{k}_{output_names_suffix}" for k, v in lulc_vars.items()
             }
             # Add the other parameters
-            for var in ["soil_ksat_vertical_factor", "h_min", "h_opt", "h_max"]:
+            for var in [
+                "soil_ksat_vertical_factor",
+                "demand_paddy_h_min",
+                "demand_paddy_h_opt",
+                "demand_paddy_h_max",
+            ]:
```

Comment:
We can then also simplify here then actually
```suggestion
            for var in [
                "soil_ksat_vertical_factor",
            ]:
```
And maybe even remove the loop.